en/docs/learn/api-security/oauth2/grant-types/implicit-grant.md

```diff
@@ -61,11 +61,23 @@ The following parameters are required to implement the Implicit grant type in W
 </tbody>
 </table>
 
-An example is given below :
-
-``` java
-    https://localhost:8243/authorize?scope=openid&response_type=id_token&redirect_uri=http://localhost:8080/playground2/oauth2client&nonce=13e2312637dg136e1&client_id=mzdQQ0RZOIqAf549ucIImB4h0SIa
-```
+!!! Tip 
+        The playground application will send a authorization request the **authorize** endpoint using the following format.
+        
+        ```java
+        https://<host>:<port>/oauth2/authorize?response_type=token
+        &client_id=<client-ID>
+        &redirect_uri=<callback-url>
+        &scope=<scope>
+        ```
```

Comment:
We can use two tabs to display the format and the example.

Reply:
fixed with b151dd4